### PR TITLE
fix: Parse attestation verdicts in run-bench

### PR DESCRIPTION
**Pull request by wbxyz**

### CODE REVIEW

File: tools/benchmark/run.go [hunk 1]
```diff
@@ -145,18 +190,12 @@ func (w *attestWorker) ProcessOne(ctx context.Context, p Package, out chan schem
...
-			// support that here.
-			verdict = schema.Verdict{
-				Target: rebuild.Target{
-					Ecosystem: rebuild.Ecosystem(p.Ecosystem),
-					Package:   p.Name,
-					Version:   v,
-				},
-				Message: "",
+			var v schema.Verdict
+			if err := json.NewDecoder(resp.Body).Decode(&v); err != nil {
```
Comment (msuozzo) [inline tools/benchmark/run.go:194]: can we use stubs here?

Comment (wbxyz) [reply tools/benchmark/run.go:194]: Yeah, but maybe in a followup PR?

Comment (wbxyz) [reply tools/benchmark/run.go:195]: Added TODO comments

Comment (msuozzo) [reply tools/benchmark/run.go:149]: eh i mean the purpose of this change is to start using the return value from the attestation verdict so it seems like the right time to move to stubs.

Comment (msuozzo) [reply tools/benchmark/run.go:149]: ok i just went ahead and did the conversion in #219. in the course of doing so, found and fixed a bug in #220

**Pull request closed without merging**